### ISSUE TITLE
Camel casing of type plural name is not behaving as before v3

### ISSUE THREAD

**Issue by kiyaGu** [neo4j/graphql#1038]
**Describe the regression**

We are trying to upgrade our systems to use v3 of this library and we are getting errors related to how type plural names are being camel cased. For example before version 3 `AWSAccount` type type was pluralised and camel cased as  `awsAccounts` but after upgrading to v3 it is now camel cased as `aWSAccounts` and we are getting the following error
```
"Cannot query field \"awsAccounts\" on type \"Query\". Did you mean \"aWSAccounts\"?"
```
we use [camelcase](https://github.com/sindresorhus/camelcase#readme) library to do the conversion on our side which used to work perfectly before V3.

**Last working version**

Worked up to version: @neo4j/graphql@2.5.5

Stopped working in version: @neo4j/graphql@3.0.0

**Type definitions**
```
type AWSAccount {
    code: String
    accountName: String
}

type DNSZone {
    awsId: String
    zoneType: String
}

```
**To Reproduce**
 Run the following query
```
{
  awsAccounts {
    code
  }
}
```
**Expected behavior**
We expect to get list of AWS accounts' code
```
{
  "data": {
    "awsAccounts": [
      {
        "code": "387243903490"
      },
      {
        "code": "295479174736"
      },
      ... ]
      }
  }    
```

**System (please complete the following information):**

-   OS: macOS
-   Node.js version: 14.16.1

**Comment by darrellwarde** [comment]
Hey @kiyaGu, with hindsight, it looks like we may have made a bit of an error here. Expect a patch (likely 3.0.1) for this soon.

**Comment by darrellwarde** [comment]
See #1040 with related test cases for examples of the proposed fix.

**Comment by darrellwarde** [comment]
This has now been fixed in 3.0.1 - do let us know if that's working for you! 🙂

**Comment by kiyaGu** [comment]
@darrellwarde it is working now. Thanks for the quick fix 🙌🏾
